aiocache/serializers/serializers.py

```diff
@@ -99,14 +99,18 @@ class PickleSerializer(BaseSerializer):
 
     DEFAULT_ENCODING = None
 
+    def __init__(self, *args, protocol=pickle.DEFAULT_PROTOCOL, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.protocol = protocol
```

Comment:
Adding ability for the PickleSerializer to have a protocol passed. By default in later versions it is 4 while in previous it is 3 which results in differences when dumping content